packages/server/lib/routes.ts

```diff
@@ -128,12 +129,16 @@ setupAuth(web);
 
 // Webapp routes (no auth).
 if (AUTH_ENABLED) {
-    web.route('/api/v1/signup').post(rateLimiterMiddleware, authController.signup.bind(authController));
-    web.route('/api/v1/signup/invite').get(rateLimiterMiddleware, authController.invitation.bind(authController));
-    web.route('/api/v1/logout').post(rateLimiterMiddleware, authController.logout.bind(authController));
-    web.route('/api/v1/signin').post(rateLimiterMiddleware, passport.authenticate('local'), authController.signin.bind(authController));
+    web.route('/api/v1/account/signup').post(rateLimiterMiddleware, signup);
+    web.route('/api/v1/account/signup/token').post(rateLimiterMiddleware, signupWithToken);
+    web.route('/api/v1/account/signup/invite').get(rateLimiterMiddleware, authController.invitation.bind(authController));
+    web.route('/api/v1/account/logout').post(rateLimiterMiddleware, authController.logout.bind(authController));
+    web.route('/api/v1/account/signin').post(rateLimiterMiddleware, passport.authenticate('local'), signin);
     web.route('/api/v1/forgot-password').put(rateLimiterMiddleware, authController.forgotPassword.bind(authController));
     web.route('/api/v1/reset-password').put(rateLimiterMiddleware, authController.resetPassword.bind(authController));
+    web.route('/api/v1/account/resend-verification-email').post(rateLimiterMiddleware, resendVerificationEmail);
+    web.route('/api/v1/account/email/:uuid').get(rateLimiterMiddleware, getEmailByUuid);
+    web.route('/api/v1/account/verify/code').post(rateLimiterMiddleware, validateEmailAndLogin);
```

Comment:
I would add those endpoints to https://github.com/NangoHQ/nango/blob/960c0e8eb7b6e6926dccf56054e9cc4fae6220f3/packages/server/lib/middleware/ratelimit.middleware.ts#L68 to limit potential abuse

Reply:
Thanks, updated. `web.route('/api/v1/account/email/:uuid')` won't work right because of the dynamic path?

Reply:
good point. let's change the condition to check the path prefix. Something like 
```    if(['/api/v1/account', '/api/v1/login', ...].some((path) => req.path.startsWith(path))) {

```

Reply:
Updated.